pydriller/repository.py

```diff
@@ -115,9 +115,17 @@ def __init__(self, path_to_repo: Union[str, List[str]],
             else set(only_commits)
             )
 
+        try:
+            if isinstance(path_to_repo, list):
+                path_to_repos = [os.fspath(path) for path in path_to_repo]
+            else:
+                path_to_repos = [os.fspath(path_to_repo)]
```

Comment:
Why don't we just transform everything in Path here instead of leaving it as strings? We could just transform it to string later when we need it, like in `_is_remote`. Wdyt?
I feel like it would be much easier to read and play with just Path, rather than a mix. But let me know since you worked on it 😄